src/Resources/config/services.yml

```diff
@@ -6,3 +6,11 @@ services:
             - "@database_connection"
         tags:
             - { name: console.command }
+
+    terminal42_leads.command.purge:
+        class: Terminal42\LeadsBundle\Command\PurgeCommand
+        arguments:
+            - "@contao.framework"
+            - "@database_connection"
+        tags:
+            - { name: console.command }
```

Comment:
Add `- { name: 'monolog.logger', channel: 'terminal42_leads' }` after adding `@logger` to the arguments.